shaka/src/util/file_system.cc

```diff
@@ -77,7 +77,7 @@ ssize_t FileSystem::FileSize(const std::string& path) const {
 }
 
 bool FileSystem::ReadFile(const std::string& path,
-                          std::vector<uint8_t>* data) const {
+                          std::string& data) const {
```

Comment:
The style guide doesn't allow non-const references; and using a `std::string` to represent bytes is usually a bad idea.  Using a `std::vector<uint8_t>` makes it clear the data produced here is binary and not necessarily text.  I don't think your changes to this file do anything, so please revert all of it.